docker/Dockerfile-py312

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2024 Bandit Authors
```

Comment:
I don't think this is necessary. We haven't previously used a copyright for the open source members.